### PR TITLE
Fix: Updated Urlscan blacklisted error message

### PR DESCRIPTION
**Pull request by vector-sec**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
The blacklisted URL error message that the Urlscan API returns is different than what the XSOAR integration is expecting. This caused the `continue_on_blacklisted_urls` argument to not have the desired effect of avoiding an error being returned to the war room. This update to the XSOAR integration restores the functionality of the `continue_on_blacklisted_urls` argument.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by YuvHayun** [comment]
Hey @vector-sec, thx for your contribution.
few small fixes:
- Please change the docker image mentioned in the yml file, the new docker image should be: 3.10.6.33415
- Please add RN, you can do it by using the command `demisto-sdk update-release-notes -i Packs/<pack name>`
- Please fix the failing unit test, my guess it's just a trailing white space or something like that.


Let me know if you have any question.

**Comment by vector-sec** [comment]
@YuvHayun I believe I've handled the docker image update and the failing unit test. For release notes I get this response from demisto-sdk when running the command

```
You are using demisto-sdk 1.7.4.
Starting to update release notes.

[WARNING]: General: [BA107] - Running on master branch while using git is ill advised.
run: 'git checkout -b NEW_BRANCH_NAME' and rerun the command.

Either no changes were found in UrlScan pack or the changes found should not be documented in the release notes file.
If relevant changes were made, please commit the changes and rerun the command.
```

File: Packs/UrlScan/Integrations/UrlScan/UrlScan_test.py [hunk 1]
```diff
@@ -36,7 +36,6 @@ def test_continue_on_blacklisted_error_arg(mocker, requests_mock, continue_on_bl
         assert return_error_mock.call_count == 0
     else:
         assert response[0].get('is_error') is True
-        assert response[0].get('error_string') == 'The submitted domain is on our blacklist, we will not scan it.'
```
Comment (YuvHayun) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan_test.py]: I think deleting this is not the solution, try to find out was is actually printed in `response[0].get('error_string')` and add it as the expected result.

**Comment by YuvHayun** [comment]
> Either no changes were found in UrlScan pack or the changes found should not be documented in the release notes file.
> If relevant changes were made, please commit the changes and rerun the command.

seems like you're working from master branch which you shouldn't do so.
I'll update the RN in the internal PR, let's just fix the unit test - we should validate the error msg.

**Review by YuvHayun — CHANGES_REQUESTED** [review]

**Comment by YuvHayun** [comment]
Hey @vector-sec.
I fixed the unit test issue - I do think it's necessary to make sure the right error msg is the one returned to the user.
Also, I added release notes to document the docker changes.

**Review by YuvHayun — APPROVED** [review]

**Comment by YuvHayun** [comment]
@vector-sec merging this pr, thx for your contribution.

**Pull request merged**